src/Index.tsx

```diff
@@ -65,25 +65,25 @@ const i18nPromise = initi18n();
   }
 
   if ($featureFlags.wishLists) {
-    saveWishListToIndexedDB();
+    store.dispatch(createWishlistObserver());
   }
-  saveAccountsToIndexedDB();
+  store.dispatch(observe(createSaveAccountsObserver()));
   store.dispatch(observe(createItemSizeObserver()));
   if ($featureFlags.themePicker) {
     store.dispatch(observe(createThemeObserver()));
   }
   store.dispatch(observe(createTilesPerCharColumnObserver()));
   setCssVariableEventListeners();
 
+  store.dispatch(observe(createSaveItemInfosObserver()));
```

Comment:
I moved this before `loadDimApiData` as it feels like it should come before it. I don't know it as well as you though so if it needs to be after just say so and I will move it back.

Reply:
I don't know that it matters either way.